package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Web of Things gateway",
   "main": "app.js",
   "scripts": {
-    "test": "./test/run-tests.sh",
+    "test": "npm run lint && npm run mocha",
     "lint": "./test/lint-all.sh",
-    "mocha": "mocha"
+    "mocha": "./test/run-tests.sh"
```

Comment:
The point of having run mocha being separate is so that we don't have to do everything that run-tests.sh.

In particular, npm test does a linting pass and a mocha pass. Having npm run mocha allows just running the mocha portion and skipping the linting portion, which can take quite a while.